system-variables.md

```diff
@@ -294,13 +294,6 @@ This variable is an alias for `last_insert_id`.
 >
 > Unlike in MySQL, the `max_execution_time` system variable currently works on all kinds of statements in TiDB, not only restricted to the `SELECT` statement. The precision of the timeout value is roughly 100ms. This means the statement might not be terminated in accurate milliseconds as you specify.
 
-### placement_checks <span class="version-mark">New in v5.3.0</span>
-
-- Scope: SESSION | GLOBAL
-- Default value: `ON`
-- This variable controls whether DDL statements validate [Placement Rules in SQL](/placement-rules-in-sql.md).
-- It is intended to be used by logical dump/restore tools to ensure that tables can always be created even if placement rules are violated. This is similar to how mysqldump writes `SET FOREIGN_KEY_CHECKS=0;` to the start of every dump file.
-
```

Comment:
This variable has been removed in TiDB 5.4. A new variable tidb_placement_mode exists instead (I will document it separately).

Reply:
Is this change will be in 5.4.1? i checked in 5.4.0, 'placement_checks' still exist, and no 'tidb_placement_mode' in system variable list.

Reply:
Here is the PR: https://github.com/pingcap/tidb/pull/31093

It looks like it was created before 5.4 code freeze, but merged after. So my understanding was not correct - it is a 6.0 only change.